### PR TITLE
add minor lodash lint rules

### PR DESCRIPTION
**Pull request by zarsky-broad**

### CODE REVIEW

File: src/libs/auth.js [hunk 1]
```diff
@@ -168,7 +168,7 @@ authStore.subscribe((state, oldState) => {
 
 export const refreshTerraProfile = async () => {
   const profile = Utils.kvArrayToObject((await Ajax().User.profile.get()).keyValuePairs)
-  authStore.update(state => _.set('profile', profile, state))
+  authStore.update(state => ({ ...state, profile }))
```
Comment (panentheos) [inline src/libs/auth.js:171]: TOL: I'm curious why you went this way and not `_.set('profile', profile)`?

**Review by panentheos — APPROVED** [review]
Looks good.

**Review by breilly2 — APPROVED** [review]
👍

File: src/libs/auth.js (hunk 1 above)
Comment (zarsky-broad) [reply src/libs/auth.js:171]: Consistency with other uses in the file.

**Pull request merged**